### ISSUE TITLE
¿Qué quieres ungir la piel?

### ISSUE THREAD

**Issue by rubereaglenest** [sarganar/I7-Spanish#21]
>ungir piel
¿Qué quieres ungir la piel ?

Creo que es una respuesta incorrecta del parser, aunque es posible que sea un problema de la construcción mía de ese verbo. No obstante lo dejo aquí como futura referencia para cuando me apetezca estudiarlo y repararlo.

Debería decir: ¿Con qué quieres ungir la piel?

O sea, creo que falla para verbos que necesitan un segundo objeto "ungir piel con aceite", "clavar punta con martillo", etc.

**Comment by rubereaglenest** [comment]
Efectivamente, es un comportamiento incorrecto de INFSP. En inglés funciona genial y sin código extra:

>anoint makke
What do you want to anoint Mákke with?

**Comment by sarganar** [comment]
No pude reproducirlo.
